WooCommerce/Classes/ViewRelated/Blaze/ProductSelector+Blaze.swift

```diff
@@ -0,0 +1,34 @@
+import Foundation
+
+internal extension ProductSelectorView.Configuration {
+    static let configurationForBlaze = ProductSelectorView.Configuration(
+        multipleSelectionEnabled: false,
+        treatsAllProductsAsSimple: true,
+        searchHeaderBackgroundColor: .listBackground,
```

Comment:
This custom background color should be removed because the default `listForeground` matches the design better.

Reply:
Handled in https://github.com/woocommerce/woocommerce-ios/commit/864f2b31551b83acef03aa79748c637d5d32e244